qiskit_metal/analyses/quantization/lumped_oscillator_model.py

```diff
@@ -55,12 +55,12 @@ class LOManalysis(QAnalysis):
     data_labels = ['lumped_oscillator', 'lumped_oscillator_all']
     """Default data labels."""
 
-    def __init__(self, design: 'QDesign', renderer_name: str = 'q3d'):
+    def __init__(self, design: 'QDesign' = None, renderer_name: str = 'q3d'):
```

Comment:
So if the user doesn't provide design or renderer_name, self.sim would be None right (line 67)? And if that were the case, then run_lom() would not work right, since line 144 would break?

Reply:
yes,
this is a problem that could be considered separate from this PR, but we can address it here as well. Probably best to address it here indeed. I will update

Reply:
Fixed in a way that does not give error.